src/promptflow/promptflow/_sdk/operations/_local_storage_operations.py

```diff
@@ -217,23 +219,26 @@ def __init__(self, run: Run, stream=False, run_mode=RunMode.Test):
         self._exception_path = self.path / LocalStorageFilenames.EXCEPTION
 
         self._dump_meta_file()
-        if run.flow:
-            try:
-                from promptflow import load_flow
+        self._eager_mode = self._calculate_eager_mode(run)
```

Comment:
NIT: we assume that the type of flow won't be changed via SDK?

Reply:
@D-W-  Please resolve this later.